src/layer/yolov3detectionoutput.cpp

```diff
@@ -115,14 +115,14 @@ static void qsort_descent_inplace(std::vector<T>& datas, std::vector<float>& sco
     if (datas.empty() || scores.empty())
         return;
 
-    qsort_descent_inplace(datas, scores, 0, scores.size() - 1);
+    qsort_descent_inplace(datas, scores, 0, static_cast<int>(scores.size() - 1));
 }
 
 static void nms_sorted_bboxes(const std::vector<BBoxRect>& bboxes, std::vector<int>& picked, float nms_threshold)
 {
     picked.clear();
 
-    const int n = bboxes.size();
+    const size_t n = bboxes.size();
 
     std::vector<float> areas(n);
     for (int i = 0; i < n; i++)
```

Comment:
signed and unsigned compare here and the following loops ?

Reply:
done